meteor/client/lib/editAttribute/edit-attribute-dropdown.tsx

```diff
@@ -73,12 +79,13 @@ const WrappedEditAttributeDropdown = wrapEditAttribute(
 		private handleChange(event) {
 			const selectedOptionValue: string = event.target.value
 			const selectedOption: DropdownOption | undefined = this.getAvailableOptions().find(
-				(option) => option.value === selectedOptionValue
+				(option) => option.value === selectedOptionValue || option.alternativeValue === selectedOptionValue
```

Comment:
Is the `alternativeValue` the label?

Reply:
No, the `alternativeValue` is primaly used for Enums to have both the string value and numeric value available.
The reason for this is necessary is because it's quite random which of the two values is used to store the Enum throughout Sofie. So the `alternativeValue` is a fail safe so we have both types available, so if we get the one that isn't used for the options, we still have it available.